src/groundlight/client.py

```diff
@@ -64,7 +64,12 @@ class Groundlight:
     POLLING_EXPONENTIAL_BACKOFF = 1.3  # This still has the nice backoff property that the max number of requests
     # is O(log(time)), but with 1.3 the guarantee is that the call will return no more than 30% late
 
-    def __init__(self, endpoint: Optional[str] = None, api_token: Optional[str] = None) -> None:
+    def __init__(
+        self,
+        endpoint: Optional[str] = None,
+        api_token: Optional[str] = None,
+        disable_tls_verification: Optional[bool] = False,
```

Comment:
```suggestion
        disable_tls_verification: Optional[bool] = None,
```
I think we should follow the same precedence logic we use for the `API_TOKEN` -- if it's set in code, use that; if it's not set in code, check the environment variable.  If we want to follow that logic, then we need the default here to be `None` so we can tell the difference between the parameter being set or not.